src/algorithms/classic/sample_based/ompl_aitstar.py

```diff
@@ -195,18 +195,10 @@ def plan(grid):
     print(x)
     print(lst)
     
-    trace=[]
-    counter = 0
-    
-    for num in lst:
-        try:
-            trace.append(Point(lst[counter],y=lst[counter + 1]))
-            counter+=2
-            print (trace)
-            print(counter)
-        except:
-            break
-
+    trace = []
+    for i in range(len(lst) - 1):
```

Comment:
I think there might be an error - previous code had `counter += 2`, so we might need to make the `for` loop iterate in steps of 2?

Reply:
Woops mb, didn't realise that.

Reply:
Fixed.